amundsen_application/static/js/utils/navigationUtils.ts

```diff
@@ -2,6 +2,7 @@ import * as qs from 'simple-query-string';
 import { createBrowserHistory } from 'history';
 
 import { ResourceType } from 'interfaces/Resources';
+import { MatchProps } from 'components/DashboardPage';
```

Comment:
nit: delete? don't see this type used here.

Reply:
Good call, it was used before.

Thanks